### PR TITLE
281 create home page

### PR DESCRIPTION
**Pull request by Kyslytsia**

### CODE REVIEW

File: client/src/components/pages/home/components/main-theme/modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export const Modal = ({ onClick }: { onClick: () => void }) => {
+  const btn = useTranslations('button');
+  const modal = useTranslations('modal');
+
+  return (
+    <div className="flex h-[180px] w-[286px] flex-col items-center justify-between rounded-[40px] border border-ebb/[.46] bg-ebb/[.63] p-[16px_24px] shadow-dashboard backdrop-blur-md tablet:h-[234px] tablet:w-[579px] tablet:p-[24px_32px] laptop:h-[282px] laptop:w-[643px] laptop:p-[32px_64px] desktop:h-[320px] desktop:w-[900px] desktop:p-[42.5px_80px]">
+      <div className="flex flex-col justify-between gap-2 tablet:gap-6">
+        <h1 className="text-center font-scada text-[24px] font-bold uppercase leading-[120%] text-dark-blue tablet:text-[32px] laptop:text-[50px] desktop:text-[64px]">
+          CHARI eRp
+        </h1>
```
Comment (ASGladchenko) [inline client/src/components/pages/home/components/main-theme/modal/index.tsx:14]: Back to upper comment. The name of Project to constants and reuse it.

File: client/src/components/pages/home/components/opportunity-chari-circle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                <card.icon className={iconWidth} />
+
+                <div className="text-center font-scada text-[12px] font-bold leading-[140%] text-black tablet:text-[16px] desktop:text-[24px]">
+                  {card.text}
+                </div>
+              </div>
+            );
+          })}
+        </div>
+      </div>
```
Comment (ASGladchenko) [inline client/src/components/pages/home/components/opportunity-chari-circle/index.tsx:47]: Logic from 22 to 43 line translate to new component

File: client/src/components/pages/home/components/reveal-on-scroll-cards/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <div className={styles.wrapper}>
+      <div className={styles.titleWrapper}>
+        <span className={styles.titleText}>{text('title.firstPart')}</span>
+        <span className={styles.titleText}>{text('title.secondPart')}</span>
+        <span className={styles.titleText}>{text('title.thirdPart')}</span>
+      </div>
+
+      {scrollItemsData(text).map((item, index) => (
+        <ScrollCard key={item.id} text={item.text} number={item.id} isEven={index % 2 === 1} />
+      ))}
```
Comment (ASGladchenko) [inline client/src/components/pages/home/components/reveal-on-scroll-cards/index.tsx:21]: Your id's is 1,2,3 you them into the key. Big chance to repeat this key. Change id or key

File: client/src/components/pages/home/components/key-advantages/accordion/styles.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+interface IStylesProps {
+  isOpen: boolean;
+  disabled?: boolean;
+}
+
+export const getStyles = ({ isOpen }: IStylesProps) => ({
+  wrapper: cn('relative w-full overflow-hidden rounded-[38px]'),
+  titleWrapper:
+    'absolute left-[1px] right-[1px] top-[1px] z-10 flex h-[74px] cursor-pointer items-center justify-between rounded-[40px] bg-white px-6',
+  title: 'font-scada text-[20px] font-bold text-lynch tablet:text-[24px] laptop:text-[26px]',
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/key-advantages/accordion/styles.ts:12]: change to standart tailwinds

File: client/src/components/pages/home/components/key-advantages/accordion/styles.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  arrow: cn(
+    'flex text-white items-center justify-center p-[6px] transition-all duration-300 rotate-[-180deg]',
+    isOpen && 'rotate-[0deg]',
+  ),
+  descriptionWrapper: cn(
+    'max-h-[76px] bg-bgAuthLinks text-sm transition-all duration-300 p-4 tablet:p-6',
+    isOpen && 'max-h-[1000px] p-[92px_16px_16px] tablet:p-[100px_24px_24px]',
+  ),
+  description: cn(
+    'font-scada text-white tablet:text-[20px] laptop:text-[24px] opacity-0 transition-all duration-300',
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/key-advantages/accordion/styles.ts:23]: changet to standart tailwinds

File: client/src/components/pages/home/components/for-ngos/index.tsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+          className="grid grid-cols-1 gap-y-[40px] tablet:grid-cols-2 tablet:gap-x-[32px] tablet:gap-y-[24px] laptop:gap-x-[42px] laptop:gap-y-[32px] desktop:laptop:gap-x-[80px] desktopXl:gap-x-[40px]"
+        >
+          {cards.map((item, index) => {
+            return (
+              <motion.div
+                key={index}
+                initial={{ opacity: 0 }}
+                animate={{ opacity: inView ? 1 : 0 }}
+                transition={{ delay: index * 0.7, duration: 0.6 }}
+                className="rounded-[20px] border border-white p-4"
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/for-ngos/index.tsx:42]: change to standart tailwinds

File: client/src/components/pages/home/components/main-theme/modal/index.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+import { useRouter } from 'next/navigation';
+
+export const Modal = () => {
+  const router = useRouter();
+  const btn = useTranslations('button');
+  const modal = useTranslations('modal');
+  const projectName = useTranslations('projectName');
+
+  return (
+    <div className="flex h-[180px] w-[286px] flex-col items-center justify-between rounded-[40px] border border-ebb/[.46] bg-ebb/[.63] p-[16px_24px] shadow-dashboard backdrop-blur-md tablet:h-[234px] tablet:w-[579px] tablet:p-[24px_32px] laptop:h-[282px] laptop:w-[643px] laptop:p-[32px_64px] desktop:h-[320px] desktop:w-[900px] desktop:p-[42.5px_80px]">
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/main-theme/modal/index.tsx:14]: Ara you sure, that use 42.5px

File: client/src/components/pages/home/components/main-theme/modal/index.tsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+      <div className="flex flex-col justify-between gap-2 tablet:gap-6">
+        <h1 className="text-center font-scada text-[24px] font-bold uppercase leading-[120%] text-dark-blue tablet:text-[32px] laptop:text-[50px] desktop:text-[64px]">
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/main-theme/modal/index.tsx:16]: change to standart tailwinds

File: client/src/components/pages/home/components/main-theme/modal/index.tsx [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+          {projectName('CHARIeRp')}
+        </h1>
+
+        <p className="text-center font-scada text-[14px] font-bold leading-[120%] text-darkGray tablet:text-[20px] laptop:text-[24px]">
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/main-theme/modal/index.tsx:20]: change to standart tailwinds

File: client/src/components/pages/home/components/opportunity-chari-circle/buttons/index.tsx [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+            key={index}
+            style={{
+              top: `${y}%`,
+              left: `${x}%`,
+            }}
+            className={`${iconWidth} absolute flex h-fit translate-x-[-50%] translate-y-[-50%] rotate-[90deg] flex-col items-center justify-center gap-2 tablet:animate-notRotate`}
+          >
+            <card.icon className={iconWidth} />
+
+            <div className="text-center font-scada text-[12px] font-bold leading-[140%] text-black tablet:text-[16px] desktop:text-[24px]">
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/opportunity-chari-circle/buttons/index.tsx:31]: change to standart tailwinds

File: client/src/components/pages/home/components/opportunity-chari-circle/index.tsx [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+import { Title } from './title';
+import { ButtonsCircle } from './buttons';
+
+export const OpportunityChariCircle = () => {
+  const text = useTranslations('homePage.opportunityChariCircle');
+
+  return (
+    <div>
+      <Title text={text('title')} className="m-auto block max-w-[202px] pb-[48px] text-[24px] tablet:hidden" />
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/opportunity-chari-circle/index.tsx:11]: change to standart tailwinds

File: client/src/components/pages/home/components/opportunity-chari-circle/index.tsx [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+      <div className="relative m-auto flex w-fit items-center justify-center">
+        <Title
+          text={text('title')}
+          className="absolute z-20 hidden tablet:block tablet:max-w-[202px] tablet:text-[32px] laptop:max-w-[227px] laptop:text-[36px] desktop:max-w-[315px] desktop:text-[50px]"
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/opportunity-chari-circle/index.tsx:16]: change to standart tailwinds

File: client/src/components/pages/home/components/reveal-on-scroll-cards/scroll-card/index.tsx [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+  const x = inView ? 0 : xWithEven;
+
+  return (
+    <div className={styles.wrapper}>
+      <motion.div
+        ref={ref}
+        transition={{ duration: 0.6 }}
+        initial={{ opacity: 0, x: xWithEven }}
+        animate={{ opacity: inView ? 1 : 0, x }}
+        className="flex w-full flex-col gap-4 rounded-[40px] bg-scrollItemBg p-[24px_32px] tablet:w-[582px] tablet:p-[32px] laptop:w-[739px] desktop:w-[911px] desktop:p-[48px] desktopXl:w-[943px] desktopXl:p-[48px_64px]"
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/reveal-on-scroll-cards/scroll-card/index.tsx:25]: change to standart tailwinds

File: client/src/components/pages/home/components/reveal-on-scroll-cards/scroll-card/index.tsx [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+      >
+        <h1 className="font-scada text-[20px] font-bold text-dark-blue tablet:text-[24px] desktop:text-[40px]">
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/reveal-on-scroll-cards/scroll-card/index.tsx:27]: change to standart tailwinds

File: client/src/components/pages/home/components/reveal-on-scroll-cards/scroll-card/styles.ts [hunk 15]
```diff
@@ -0,0 +1,9 @@
+import { cn } from '@/utils';
+
+export const getStyles = (isEven: boolean) => ({
+  wrapper: cn('flex ', isEven ? 'justify-end' : 'justify-start'),
+  text: cn(
+    'font-roboto uppercase leading-[150%] text-superBlue tablet:text-[20px] desktop:text-[24px]',
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/reveal-on-scroll-cards/scroll-card/styles.ts:6]: change to standart tailwinds

File: client/src/components/pages/home/components/reveal-on-scroll-cards/styles.ts [hunk 16]
```diff
@@ -0,0 +1,7 @@
+export const getStyles = () => ({
+  wrapper:
+    'flex w-full flex-col gap-6 tablet:gap-[56px] laptop:gap-16 desktopXl:gap-24 px-4 tablet:px-8 desktop:px-[136px] desktopXl:px-[86px]',
+  titleWrapper: 'flex flex-col mx-auto',
+  titleText:
+    'text-center font-scada text-[24px] leading-[120%] font-bold uppercase text-dark-blue tablet:text-[32px] laptop:text-[36px] desktop:text-[50px]',
```
Comment (Wolshebnik) [inline client/src/components/pages/home/components/reveal-on-scroll-cards/styles.ts:6]: change to standart tailwinds

**Review by Wolshebnik — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**